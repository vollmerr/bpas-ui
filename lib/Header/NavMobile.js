var _templateObject = _taggedTemplateLiteral(['\n  color: ', ';\n  cursor: pointer;\n  float: right;\n  margin-top: 1.2em;\n\n  &:hover {\n    text-decoration: none;\n  }\n'], ['\n  color: ', ';\n  cursor: pointer;\n  float: right;\n  margin-top: 1.2em;\n\n  &:hover {\n    text-decoration: none;\n  }\n']);

function _taggedTemplateLiteral(strings, raw) { return Object.freeze(Object.defineProperties(strings, { raw: { value: Object.freeze(raw) } })); }

import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
import theme from '../../util/theme';
import Icon from '../Icon';

var Menu = styled.a(_templateObject, theme.color.theme.primary);

/** Renders the navbar logo  */
function NavMobile(_ref) {
  var onClick = _ref.onClick;

  return React.createElement(
    Menu,
    { onClick: onClick },
    React.createElement(Icon, { icon: 'menu', size: 36 })
  );
}

NavMobile.PropTypes = {
  onClick: PropTypes.func.isRequired
};

export default NavMobile;