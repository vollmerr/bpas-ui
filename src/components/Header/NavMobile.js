import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
import theme from '../../util/theme';
import Icon from '../Icon';


const Menu = styled.a`
  color: ${theme.color.primary};
  cursor: pointer;
  float: right;
  margin-top: 1.2em;

  &:hover {
    text-decoration: none;
  }
`;

/** Renders the navbar logo  */
function NavMobile({
  onClick,
}) {
  return (
    <Menu onClick={onClick} >
      <Icon icon={'menu'} size={36} />
    </Menu>
  );
}

NavMobile.PropTypes = {
  onClick: PropTypes.func.isRequired,
};

export default NavMobile;
