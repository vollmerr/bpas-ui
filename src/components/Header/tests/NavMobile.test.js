import React from 'react';
import { shallow } from 'enzyme';
import NavMobile from '../NavMobile';

describe('<NavMobile />', () => {
  it('renders correctly', () => {
    const tree = shallow(
      <NavMobile />
    );
    expect(tree).toMatchSnapshot();
  });

  xit('passes the correct props down', () => {});
});
