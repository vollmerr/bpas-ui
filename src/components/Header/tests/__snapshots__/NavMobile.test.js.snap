// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavMobile /> renders correctly 1`] = `
<styled.a>
  <Icon
    icon="menu"
    size={36}
  />
</styled.a>
`;
